cmd/utils.go

```diff
@@ -805,10 +829,10 @@ func checkDockerImageExistsLocally(imageToPull string) bool {
 	return false
 }
 
-//dockerPullImage
-// pulls docker image, if APPSODY_PULL_POLICY set to IFNOTPRESENT
+//containerPullImage
+// pulls buildah / docker image, if APPSODY_PULL_POLICY set to IFNOTPRESENT
 //it checks for image in local repo and pulls if not in the repo
-func dockerPullImage(imageToPull string) error {
+func containerPullImage(imageToPull string, buildah bool) error {
```

Comment:
I'm not sure this is the best name - are we pulling a container or an image?  We're pulling an image, so just call it `pullImage()`?

Reply:
In general, the PR says "container" a lot when it means "image".

Reply:
@ianpartridge - I tend to agree; was just retaining the original wordings with generalizations. If you have strong opinions I could do a find-replace (there are other tidy-ups btw) , or else let it pass for now, so that the waiting pipeline changes can move forward, and then revisit later? thanks.

Reply:
I generally feel strongly that we should ship excellent code 😁

Reply:
@ianpartridge - here you go! made those naming changes, ptal.